state_machine/drivers/pycubedmini/lib/pycubed.py

```diff
@@ -45,611 +233,421 @@ class Satellite:
-        self.logfile = "/sd/logs/log000.txt"
+    def _init_i2c2(self):
+        """ Initialize I2C2 bus """
+        try:
+            self._i2c2 = busio.I2C(board.SCL2, board.SDA2)
+            self.hardware['I2C2'] = True
+        except Exception as e:
+            print("[ERROR][Initializing I2C2]", e)
 
-        # Define radio
-        self._rf_cs = digitalio.DigitalInOut(board.RF_CS)
-        self._rf_rst = digitalio.DigitalInOut(board.RF_RST)
-        self.radio_DIO0 = digitalio.DigitalInOut(board.RF_IO0)
-        self.radio_DIO0.switch_to_input()
-        self.radio_DIO1 = digitalio.DigitalInOut(board.RF_IO1)
-        self.radio_DIO1.switch_to_input()
-        self._rf_cs.switch_to_output(value=True)
-        self._rf_rst.switch_to_output(value=True)
+    def _init_i2c3(self):
+        """ Initialize I2C3 bus """
+        try:
+            self._i2c3 = busio.I2C(board.SCL3, board.SDA3)
+            self.hardware['I2C3'] = True
+        except Exception as e:
+            print("[ERROR][Initializing I2C3]", e)
 
-        # Initialize Neopixel
+    def _init_spi(self):
+        """ Initialize SPI bus """
         try:
-            self.neopixel = neopixel.NeoPixel(
-                board.NEOPIXEL, 1, brightness=0.2, pixel_order=neopixel.GRB)
-            self.neopixel[0] = (0, 0, 0)
-            self.hardware['Neopixel'] = True
+            self._spi = busio.SPI(board.SCK, MOSI=board.MOSI, MISO=board.MISO)
+            self.hardware['SPI'] = True
         except Exception as e:
-            print('[WARNING][Neopixel]', e)
+            print("[ERROR][Initializing SPI]", e)
 
-        # Initialize sdcard
+    def _init_sdcard(self):
+        """ Define SD Parameters and initialize SD Card """
         try:
             self._sd = sdcardio.SDCard(self.spi, board.CS_SD, baudrate=4000000)
-            self._vfs = storage.VfsFat(self._sd)
+            self._vfs = storage.VfsFat(self.sd)
             storage.mount(self._vfs, "/sd")
             sys.path.append("/sd")
             self.hardware['SDcard'] = True
-            # self.new_log() # create new log file
         except Exception as e:
-            print('[ERROR][SD Card]', e)
+            print('[ERROR][Initializing SD Card]', e)
 
-        # Initialize radio - UHF
+    def _init_neopixel(self):
+        """ Define neopixel parameters and initialize """
         try:
-            self.radio = pycubed_rfm9x.RFM9x(
-                self.spi, self._rf_cs, self._rf_rst,
-                self.UHF_FREQ, rfm95pw=True)
-            self.radio.dio0 = self.radio_DIO0
-            self.radio.sleep()
-            self.hardware['Radio'] = True
+            self._neopixel = neopixel.NeoPixel(
+                board.NEOPIXEL, 1, brightness=0.2, pixel_order=neopixel.GRB)
+            self.neopixel[0] = (0, 0, 0)
+            self.hardware['Neopixel'] = True
         except Exception as e:
-            print('[ERROR][RADIO]', e)
+            print('[WARNING][Initializing Neopixel]', e)
```

Comment:
That's the way it was in the old driver; my best guess is that it's because Neopixel initialization failure isn't fatal to the cubesat's operation, but other initialization failures are. Either way, I can change it to ERROR if that makes more sense